src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -112,7 +112,7 @@ private LauncherHelper() {}
 
     private static StringBuilder outBuf = new StringBuilder();
 
-    private static final String INDENT = "    ";
+    private static final String INDENT = " ".repeat(4);
```

Comment:
Revert this, its just making a simple constant into a runtime expression.

Reply:
Thanks Roger. Fair point. Done.